### PR TITLE
Add meson.build to release howto and replace IRC by Matrix

### PR DESCRIPTION
**Pull request by eht16**

### CODE REVIEW

File: doc/making-a-release [hunk 1]
```diff
@@ -24,7 +24,8 @@ version number (e.g. `/home/geany/publish-release 1.36`).
 Make sure the public part of the GPG key used to sign the archives
 can be found on the download page.
 
-Windows builds (enrico?).
+Windows builds are created mostly by Enrico following the instructions on
```
Comment (elextr) [inline doc/making-a-release:27]: Where do I download the "Enrico" application, or can one be distributed with each Geany? :grin:

**Comment by elextr** [comment]
Other than one smartass comment :-D, LGBI

File: doc/making-a-release (hunk 1 above)
Comment (eht16) [reply doc/making-a-release:27]: Ha, I wouldn't recommend that. It's already a few decades around but still buggy from time to time. Also it needs lots of resources, mainly water and food to work properly and somtimes even do weird output.

Not sure I would recommend this application.

Comment (eht16) [reply doc/making-a-release:27]: On a more serious note, I hope we will update this part again in the future.

Recently I had to solve the first quirk with running current MSYS2 packages on my old Windows 7 VM which is not supported anymore.
As stated before, I will *not* upgrade to any newer Windows version ever.

So, either we find another lost soul to have fun with Windows or we use the CI build scripts to create release builds (which is entirely possible already).

**Review by b4n — APPROVED** [review]
LGTM

**Pull request merged**